actions/class.TestRunner.php

```diff
@@ -768,9 +768,8 @@ public function storeItemVariableSet()
             $session = $this->getTestSession();
             $state = $session->getState();
             if($state == AssessmentTestSessionState::SUSPENDED){
-                if (\oat\oatbox\service\ServiceManager::getServiceManager()->has('taoProctoring/delivery')) {
-                    $deliveryService = \oat\oatbox\service\ServiceManager::getServiceManager()->get('taoProctoring/delivery');
-                    $deliveryService->setHasBeenPaused($session->getSessionId(), true);
+                if (common_ext_ExtensionsManager::singleton()->isEnabled('taoProctoring')) {
```

Comment:
This functionality has been implemented not by me. I just trying to merge conflicts in my PR's to make them up to date.